packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -322,6 +322,7 @@ export const announceAuthService = ({
   router: VueRouter
 }): void => {
   authService.initialize(configurationManager, clientService, store, router)
+  vue.prototype.$authService = authService
   set(vue, '$authService', authService)
```

Comment:
I guess you can remove this line then.

Reply:
Thought so too, but it gives me: `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'initializeContext')` 🤔  It's the same for `$permissionManager`.

Reply:
Because of this here: https://github.com/owncloud/web/blob/fix-missing-public-password-form/packages/web-runtime/src/router/setupAuthGuard.ts#L14

Reply:
What a mess. You're a hero!

Reply:
👏